astropy/visualization/wcsaxes/core.py

```diff
@@ -729,6 +729,34 @@ def tick_params(self, axis='both', **kwargs):
                 if spine in coord.axislabels.get_visible_axes():
                     coord.tick_params(**kwargs)
 
+    def set_world_aspect(self, aspect, **kwargs):
+        """
+        Set the aspect of the axes in world coordinates.
```

Comment:
I think it would be worth adding a notes section here saying how this is calculated, and what it's limitations are, because for some projections doing this at the 0th pixel is going to be outright weird.